based on user feedback (#556)
SAP-samples/kyma-runtime-extension-samples: from-zero-to-cap-on-kyma/README.md

## Prerequisites

- [SAP BTP, Kyma runtime instance](../prerequisites/README.md#kyma)

>    > Note: if you are using BTP trial account, use trial subaccount that supports SAP Hana. At the time of writing SAP Hana is available in the US, but not in Signapore

Reviewer: ```suggestion

> [!Note]
> If you're using an SAP BTP trial account, use a subaccount that supports SAP Hana Cloud. At the time of creating the sample (June 2025), SAP Hana Cloud is available in the US, but not in Singapore.
```
Is it SAP Hana or SAP Hana Cloud?

- [Docker](../prerequisites/README.md#docker)
- [make](https://www.gnu.org/software/make/)
- [Kubernetes tooling](../prerequisites/README.md#kubernetes)
- [Pack](../prerequisites/README.md#pack)
- [NodeJS 20 or higher](https://nodejs.org/en/download/)
- [SAP CAP](../prerequisites/README.md#sap-cap)
- SAP Hana Cloud Instance

>    > Note: If using trial, make sure your subaccount location supports SAP Hana Cloud

Reviewer: ```suggestion

> [!Note]
> If you're using an SAP BTP trial account, make sure your subaccount location supports SAP Hana Cloud.
```

### Configuring Environment Variables

1. Set up the required environment variables:

>    > Note: You will download the kubeconfig file from your Kyma runtime instance in the SAP BTP cockpit. or, if you are using default, it might be available in your local machine under `~/.kube/config`.

Reviewer: ```suggestion
> [!Note] 
> You can download the kubeconfig file from your Kyma runtime instance in the SAP BTP cockpit. If you are using default, you should also be able to access the kubeconfig from your local machine under `~/.kube/config`.
```
Default what?

`cds` can intelligently inspect what is defined in your CAP application and generate the necessary configurations (Helm charts) to deploy the application on Kyma runtime.

> 1. Create a Helm chart. When asked for the registry server, if you are using docker hub, enter your use name. For private registries, enter the registry server URL.

Reviewer: ```suggestion
1. Create a Helm chart. When asked for the registry server:
   * If you're using Docker Hub, enter your username. 
   * If you're using a private registry, enter the registry server URL.
```
